src/fragmentarium/ui/fragment/TeiExport.js

```diff
@@ -0,0 +1,138 @@
+export function teiExport(fragment) {
+  var res = ''
```

Comment:
Use descriptive and unshortened names. I.e `result`.

Reply:
changed res to result